packages/form-js-editor/src/features/properties-panel/entries/DefaultValueEntry.js

```diff
@@ -29,7 +29,11 @@ export default function DefaultOptionEntry(props) {
   const entries = [];
 
   // Only make default values available when they are statically defined
-  if (!INPUTS.includes(type) || VALUES_INPUTS.includes(type) && !field.values) {
+  if (
+    !INPUTS.includes(type) &&
+    type !== 'image' ||
```

Comment:
This is one of the symptoms of those intermingling concepts I was referring to in my large initial point on `defaultValue`, it's codebase structure I think we shouldn't try to hack our way around. Value, defaultValue ect are all related to the lifecycle of form field state which image is not.

Note that `Text` doesn't need an exception, because it creates its own property (text) to represent its value. I think we should do the same.

Reply:
As proposed, let's ditch that one completely 👍